scripts/remote-heal/heal_for_heartbeat.yml

```diff
@@ -0,0 +1,84 @@
+---
+#prybook will try heal for the hear.beat.ping alert
+# * oso-moniotoring pod dead (restarted)
+# * atomic-node service dead（restarted)
+# * host lost connection (try to restart the instance)
+# * if this is a zag pod , delete it from the zabbix
+
+
+- name: check if the this is zag pod
+  hosts: localhost
+
+  pre_tasks:
+    - name: Ensure needed variables are set
+      fail:
+        msg: "Value {{ item }} is not set!"
+      when: "{{ item }} is undefined"
+      with_items:
+      - cli_nodename
+
+    - name: check if the alert is a zag pod
+      set_fact :
+        is_zagg_pod : true
+      when: cli_nodename | search("oso-rhel7-zagg-web")
+
+
+    - name: make sure openshift-tools-ansible-zabbix is installed 
```

Comment:
doesn't the fact that this file is on the filesystem already mean that openshift-tools-ansible-zabbix is installed (b/c of the RPM dependencies)?